src/relay/ir/pretty_printer.cc

```diff
@@ -304,14 +304,16 @@ class PrettyPrinter :
    * \return The corresponding name.
    */
   Doc AllocTypeVar(const TypeVar& var) {
+    if (memo_type_.count(var)) {
+      Doc val = memo_type_[var];
+      val << "-malformed-ir";
```

Comment:
Could we change this to print in a more informative way? for example maybe use a colored highlight to show which part is malformed?

Reply:
there is no color highlighting in the current doc.